### PR TITLE
RDM-6328 : Functional Tests for 'Get the pagination metadata for a case data search for Citizen' operation

### PR DESCRIPTION
**Pull request by aashishaggarwal123**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-6328

### Change description ###
RDM-6328 : Implement Functional Tests for 'Get the pagination metadata for a case data search for Citizen' operation

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/aat/resources/features/F-057/F-057.feature [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    #And the request [does not provide valid authorized access to the operation]
+    Given it is submitted to call the [Get the pagination metadata for a case data search for Citizen] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [has the 403 return code]
+    And the response has all the details as expected
+
+  @S-140
+  Scenario: must return pagination metadata successfully for correct inputs
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-057/F-057.feature]: Happy path scenarios always on top, please.

File: src/aat/resources/features/F-057/F-057.feature [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    Given it is submitted to call the [Get the pagination metadata for a case data search for Citizen] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [has the 400 return code]
+    And the response has all the details as expected
+
+  @S-522
+  Scenario: must return 400 when security classification in input parameters is present and invalid
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    #And the request [does not provide valid security classification in input parameters]
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-057/F-057.feature]: Please activate this line in all occurrences. We will need to get the relevant code merged from DSL-2 implementation.

**Comment by aashishaggarwal123** [comment]
Hi Mutlu, I have updated the feature file as per comments. Kindly check and approve.

**Comment by aashishaggarwal123** [comment]
Created a new PR 662 for this JIRA.

**Pull request closed without merging**